### ISSUE TITLE
No chat on VODs

### ISSUE THREAD

**Issue by yoimlive777** [worldreboot/twitch-reloaded-roku#19]
I haven't tried watching a stream live but I noticed there's no chat (I enabled it in settings) for VODs. Might want to add that.

Also an ability to see the BTTV emotes would be cool (not sure if possible)

**Comment by worldreboot** [comment]
Yes, chat for for VODs would be nice. Also, BTTV emotes should already be enabled (except animated GIFs)
